src/node_modules/app/elasticsearch/typings/es.ts

```diff
@@ -0,0 +1,20 @@
+export type Opaque<T, K> = T & { __opaque__: K };
+
+export type date = Opaque<Date, 'date'>;
+export type float = Opaque<number, 'float'>;
+export type integer = Opaque<number, 'integer'>;
+
+export type Date_YYYYMMDD = Opaque<string, 'Date_YYYYMMDD'>;
+export type Date_YYYYMMDD_dash = Opaque<string, 'Date_YYYYMMDD_dash'>;
+export type Date_YYYYMMDD_dash_time = Opaque<string, 'Date_YYYYMMDD_dash_time'>;
+export type Date_YYYYMMDD_dash_T_time = Opaque<string, 'Date_YYYYMMDD_dash_T_time'>;
+
+export type ES_geo_point_obj = {
+    lat: float,
```

Comment:
[1] minor, I'd replace these 4 blanks with tabs, and the same in `dsl.ts`... -> see [2]